### ISSUE TITLE
ResizeWindow does not work with Puppeteer 1.5

### ISSUE THREAD

**Issue by xt1** [Codeception/CodeceptJS#1112]
#### What are you trying to achieve?

Resize the window using the Puppeteer helper.
With Puppeteer 1.5 the test fails.
If i `npm i puppeteer@1.4` then the test succeeds.

Not sure if this is a bug in Codecept or in Puppeteer. 
Starting here.

#### What do you get instead?

```bash
> node_modules\.bin\codeceptjs.cmd run --verbose                                                                       
CodeceptJS v1.2.1                                                                                                      
Using test root "D:\Seven\Patches\Patch80\Clients\SM.web\Tests\codecept"                                               
                                                                                                                       
Puppeteer broken --                                                                                                    
   [1] Starting recording promises                                                                                     
   Emitted | suite.before ([object Object])                                                                            
 > [Session] Starting singleton browser session                                                                        
 Make the window the right size                                                                                        
   Emitted | test.before ([object Object])                                                                             
   Emitted | test.start ([object Object])                                                                              
   Emitted | step.before (I am on page "/")                                                                            
   Emitted | step.after (I am on page "/")                                                                             
   Emitted | step.before (I resize window 1200, 800)                                                                   
   Emitted | step.after (I resize window 1200, 800)                                                                    
   Emitted | step.start (I am on page "/")                                                                             
 * I am on page "/"                                                                                                    
 > [Url] http://localhost/CRM.web80/Security/Login.aspx?ReturnUrl=%2fCRM.web80%2f                                      
 > [verbose]                                                                                                           
   Emitted | step.passed (I am on page "/")                                                                            
   Step finished in 2.409 sec                                                                                          
   Emitted | step.finish (I am on page "/")                                                                            
   Emitted | step.start (I resize window 1200, 800)                                                                    
 * I resize window 1200, 800                                                                                           
   [1] Error | Error: Protocol error (Browser.setWindowBounds): Browser window not found undefined                     
   Emitted | step.failed (I resize window 1200, 800)                                                                   
   Step finished in 0.001 sec                                                                                          
   Emitted | step.finish (I resize window 1200, 800)                                                                   
   [1] Error | Error: Protocol error (Browser.setWindowBounds): Browser window not found undefined                     
   [1] Starting <teardown> session                                                                                     
   Emitted | test.failed ([object Object])                                                                             
   Emitted | test.finish ([object Object])                                                                             
   [1] <teardown> Stopping recording promises                                                                          
 > Screenshot is saving to D:\Seven\Patches\Patch80\Clients\SM.web\Tests\codecept\output\Make_the_window_the_right_size
.failed.png                                                                                                            
 × FAILED in 2711ms      

-- FAILURES:

  1) Puppeteer broken
       Make the window the right size:
     Protocol error (Browser.setWindowBounds): Browser window not found undefined

  Scenario Steps:

  - I.resizeWindow(1200, 800) at Test.Scenario (proj_test.js:6:7)
  - I.amOnPage("/") at Test.Scenario (proj_test.js:5:7)

  Error: Protocol error (Browser.setWindowBounds): Browser window not found undefined
      at Promise (node_modules\puppeteer\lib\Connection.js:86:56)
      at new Promise (<anonymous>)
      at Connection.send (node_modules\puppeteer\lib\Connection.js:85:12)
      at Puppeteer.resizeWindow (node_modules\codeceptjs\lib\helper\Puppeteer.js:403:36)
      at Step.run (node_modules\codeceptjs\lib\step.js:26:47)
      at recorder.add (node_modules\codeceptjs\lib\actor.js:68:23)
      at <anonymous>
                                                                                              
```

Test source code 

```js
Scenario('Make the window the right size', (I) => {
    I.amOnPage('/');   
    I.resizeWindow(1200,800);
});
```

### Details

* CodeceptJS version:    1.2.1
* NodeJS Version: v8.11.2
* Operating System: Win10 64bit 10.0.15063
* Puppeteer 1.5.0
* Configuration file:

```js
{
  "tests": "./*_test.js",
  "timeout": 10000,
  "output": "./output",
  "helpers": {
    "Puppeteer": {
      "url": "http://localhost/CRM.web80/",
      "show": true,
      "restart": false,
      "waitForTimeout": 2000,
      "waitForAction": 1000
    }
  },
  "include": {
    "I": "./steps_file.js"
  },
  "bootstrap": false,
  "mocha": {},
  "name": "codecept"
}
```

**Comment by tranquille** [comment]
Hi, I have the same problem on Mac...
NodeJS: v8.11.3
CodeceptJS: 1.2.1
Puppeteer:  1.5.0
Operating System: Mac OS 10.12.6

**Comment by sergejkaravajnij** [comment]
I have the same.
I've commented workaround founded in
`node_modules/codeceptjs/lib/helper/Puppeteer.js:402`

After that the issue gone.
So, I believe this workaround not required anymore.

**Comment by artigat1** [comment]
I just upgraded to 1.5.0 and am getting the same problem.

NodeJS: 9.11.1
CodeceptJS: 1.2.1
Puppeteer: 1.5.0
Operating System: Windows 10

**Comment by DavertMik** [comment]
Fixed in #1122
